app/src/main/java/com/locked/shingranicommunity/event/EventCreateViewModel.kt

```diff
@@ -55,15 +61,28 @@ class EventCreateViewModel @Inject constructor(
         }
     }
 
+    fun showDatePicker(v: View) {
+        val c = Calendar.getInstance()
+        val dialog = DatePickerDialog(v.context, DatePickerDialog.OnDateSetListener { _, year, month, dayOfMonth ->
+            val _year = year.toString()
+            val _month = if (month + 1 < 10) "0" + (month + 1) else
+                (month + 1).toString()
+            val _date = if (dayOfMonth < 10) "0$dayOfMonth" else dayOfMonth.toString()
+            date.value = "$_year-$_month-$_date"
+        }, c[Calendar.YEAR], c[Calendar.MONTH], c[Calendar.MONTH])
+        dialog.datePicker.minDate = System.currentTimeMillis() - 1000
+        dialog.show()
+    }
+
```

Comment:
move this code to `EventCreateFragment` and use it the way you use `showTimePicker()` function